### PR TITLE
[Messages] Misc UI fixes and space key remapping

### PR DESCRIPTION
**Pull request by l2succes**
Closes: https://github.com/artsy/emission/issues/753

Misc UI fixes:

- It fixes the UI layout of `ConversationSnippet` component as the rows are off by a few pixels

Before:

After:

- It also fixes the separator styling in the message component, we were missing the padding around it and the last row should not have a separator under it.

- It also moves the loading indicator inside of the messages list view so that it doesn't always appear if the user is scrolling slower than the time needed to load the next page.

It also includes a re-bind of the `space` key in Emission. As of now, we aren't able to hit space as it's used as a toggle for hiding/showing the black back button in the simulator. After this PR, the binding will `ctrl+space` and `space` will return to its default binding.

This shows a loading indicator inside the list view when a user scrolls up fast.

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Inbox/Typography/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ###### Dev
 
+-   Remaps space key bidding to ctrl+space so we can enter spaces fromthe simulator - luc
```
Comment (mzikherman) [inline CHANGELOG.md]: looks like youre.....missing a space in **fromthe** 😎

File: src/lib/Components/Inbox/Conversations/Message.tsx [hunk 2]
```diff
@@ -52,6 +57,11 @@ const TimeStamp = styled(MetadataText)`
   font-size: 11.5
 `
 
+const Seperator = styled(DottedLine)`
```
Comment (mzikherman) [inline src/lib/Components/Inbox/Conversations/Message.tsx]: typo: ***Separator**

File: src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const CardContent = styled(HorizontalLayout)`
 
 const TextPreview = styled(VerticalLayout)`
   margin-left: 15;
-  margin-bottom: 3;
+  margin-bottom: 15;
```
Comment (mennenia) [inline src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx]: 🙏

**Comment by l2succes** [comment]
Sounds good

**Pull request merged**